core/types/transaction_signing.go

```diff
@@ -42,15 +42,7 @@ type sigCache struct {
 // MakeSigner returns a Signer based on the given chain config and block number.
 func MakeSigner(config *params.ChainConfig, blockNumber *big.Int) Signer {
 	var signer Signer
-	switch {
-	// TODO (chao) clean up different forks in ETH code
-	case config.IsEIP155(blockNumber):
-		signer = NewEIP155Signer(config.ChainID)
-	case config.IsHomestead(blockNumber):
-		signer = HomesteadSigner{}
-	default:
-		signer = FrontierSigner{}
-	}
+	signer = NewEIP155Signer(config.ChainID)
```

Comment:
This is the most current signing method for ethereum?

Reply:
I think so. Also, the homestead signing method not work in our code.

Reply:
ok. EIP155 fork is at "EIP155Block:         big.NewInt(2675000)," which means the 1 transaction already in our mainnet is using default Frontier signer. Will these two compatible? If not, we need to add the fork logic.

Reply:
Don't know if compatible with Frontier signer, I can do some test about it

Reply:
Local test not work for Frontier and Homestead signer